client/starwhale/base/cloud.py

```diff
@@ -235,23 +235,31 @@ def _fetch_bundle_all_list(
         uri_typ: str,
         page: int = DEFAULT_PAGE_IDX,
         size: int = DEFAULT_PAGE_SIZE,
+        _filter: t.Dict[str, t.Any] = {},
     ) -> t.Tuple[t.Dict[str, t.Any], t.Dict[str, t.Any]]:
+        _params = {"pageNum": page, "pageSize": size}
+        _params.update(_filter)
         r = self.do_http_request(
             f"/project/{project_uri.project}/{uri_typ}",
-            params={"pageNum": page, "pageSize": size},
+            params=_params,
             instance_uri=project_uri,
         ).json()
-
         objects = {}
 
+        _page = page
+        _size = size
+        if _filter.get("latest", False):
```

Comment:
[only for cloud] the api of 'list' are the collection for all runtime's latest info,  so the filter of 'latest' is no use in here in actual?